scipy/signal/signaltools.py

```diff
@@ -361,34 +366,59 @@ def fftconvolve(in1, in2, mode="full"):
     in1 = asarray(in1)
     in2 = asarray(in2)
 
+    if axes is None:
+        pass  # skip next few checks
+    elif not axes:
+        raise ValueError('if axes is specified it must be non-empty')
+    elif min(axes) < 0:
```

Comment:
This can cause an error for bad arguments like `axes=True` or even `axes=-1`, then `min` will fail to iterate properly. Maybe an `axes = np.atleast_1d(axes)` followed by `ndim == 1` and int-`dtype` check would be more robust? It might be worth looking at how multiple-axis iteration is done elsewhere in SciPy and refactoring, if necessary, into some function that handles these in a sensible, uniform way.